using-kontena/stack-file.md

```diff
@@ -158,6 +158,7 @@ There are many configuration options available for defining Kontena Services. In
 * **`build`** - Specify the configuration options that are applied at docker image build time. [Learn more](https://docs.docker.com/compose/compose-file/compose-file-v2/#build). **NOTE!** The `network` and `label` options mentioned in docker compose documentation do not work with Kontena Stack Files.
 * **`cap_add`** and **`cap_drop`** - Add or drop container capabilities. [Learn more](https://docs.docker.com/compose/compose-file/compose-file-v2/#cap_add-cap_drop).
 * **`command`** - Recall the optional COMMAND. [Learn more](https://docs.docker.com/compose/compose-file/compose-file-v2/#command).
+* **`cpus`** - Specify how much of the available CPU resources a Kontena Service Instance can use. For example, if the host machine has two CPUs and you set `cpus: 1.5`, the Kontena Service Instance will be guaranteed to be able to access at most one and a half of the CPUs. Kontena Platform will automatically target deployment to nodes that have at least number of specified CPUs available.
```

Comment:
Careful with the wording around `guaranteed ... at most` here, it's liable to be misunderstood unless reading very carefully. I suggest something using `limited`:

> The Kontena Service Instance will be limited to utilize at most one and a half CPUs. 

Specifically, `cpus: 1.5` does ***NOT*** *guarantee* that the service will get 1.5 CPUs...